src/script/chem/struct/fragment.js

```diff
@@ -0,0 +1,68 @@
+/****************************************************************************
+ * Copyright 2018 EPAM Systems
```

Comment:
Set Copyright to 2020 for new files

Reply:
Do you mean for new files in this branch or new files created by me?

Reply:
All files in branch

Reply:
Done